lambdatimes/src/components/Content/Content.js

```diff
@@ -17,38 +17,26 @@ export default class Content extends Component {
   }
 
   componentDidMount() {
-    // Once the component has mounted, get the data and reflect that data on the state.
+    this.setState({tabs: tabData})
+    this.setState({cards: cardData})
```

Comment:
You can put both of these into one setState like so, this.setState({ tabs: tabData, cards: cardData})